docs/control-structures.rst

```diff
@@ -117,8 +162,9 @@ External Function Calls
 The expressions ``this.g(8);`` and ``c.g(2);`` (where ``c`` is a contract
 instance) are also valid function calls, but this time, the function
 will be called "externally", via a message call and not directly via jumps.
-Please note that function calls on ``this`` cannot be used in the constructor, as the
-actual contract has not been created yet.
+
+...note::
+     You cannot use the function calls on ``this`` in the constructor, as the actual contract has not been created yet.
```

Comment:
I would remove "the" from "cannot use the function calls" - not sure about it, though.

Reply:
No you're right :)